23. 2blog_2news.md

```diff
@@ -0,0 +1,14 @@
+
+23. Lấy 2 blog, 2 news mà user có id = 1 đã comment
+
+```mysql
+SELECT blog.id,blog.title,id_blog.target_table FROM blog 
+  JOIN (SELECT target_id, target_table FROM `comment` 
+        WHERE user_id = 1 && target_table = 'blog' LIMIT 2) id_blog  
+  ON blog.id = id_blog.target_id 
```

Comment:
Điều kiện chỗ JOIN comment ON (comment.target_table = 'blog' AND comment.target_id = blog.id AND comment.user_id = 1 LIMIT 2)
không phải dùng subquery nữa cho nhẹ performance. Câu dưới cũng tượng tự